spec/setup_test_database.rb

```diff
@@ -7,5 +7,6 @@ def setup_test_database
 
 def add_row_to_test_database
   connection = PG.connect(dbname: 'chitter_test')
```

Comment:
OK so, the method now adds two rows but to know that I have to come here and read it. Maybe the name of the method should be add_two_rows_to_test_db or you could make it more flexible by allowing the method to take an arg (the post body) and then call it twice in your test. This last version would make the tests a little easier to follow.